devel/nvm/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nvm-sh nvm 0.35.2 v
+
+categories          devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+maintainers         @FranklinYu openmaintainer
+description         Node version manager
+long_description    NVM is a simple shell script to manage multiple active Node.js versions.
+
+checksums           rmd160  f4e76639c4c52a8855ef93481eee5f5a0e874d3f \
+                    sha256  f1369519f9eb36ffba2393ddb5829844c7c4d941304934282d0f43d39f176f39 \
+                    size    114181
+
+use_configure       no
+default_variants    +bash_completion
```

Comment:
I think it might be fine, given how tiny the bash completion files are, to simply include that in the main port and not have a variant.